client/components/Index/SignInModal.jsx

```diff
@@ -17,23 +21,34 @@ class SignInModal extends React.Component {
     this.onChange = this.onChange.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
   }
+  /**
+   * @return {null} Triggers the signinRequest action on click of submit button
+   * @param {e} e
+   */
   onSubmit(e) {
     this.setState({ errors: {} });
     e.preventDefault();
     this.props.signinRequest(this.state).then(
       () => {
 
       },
-      (errors) => {
-        console.log(errors);
+      () => {
+        // console.log(errors);
       }
     );
   }
+  /**
+   * @return {null} Updates the state as the user types into the input the fields
+   * @param {e} e
+   */
   onChange(e) {
     const state = this.state;
```

Comment:
This does not successfully copy an object. The 'const' variable called state still keeps a pointer to the original this.state value.